src/industry.h

```diff
@@ -60,20 +60,38 @@ DECLARE_ENUM_AS_BIT_SET(IndustryControlFlags);
  * Defines the internal data of a functional industry.
  */
 struct Industry : IndustryPool::PoolItem<&_industry_pool> {
+	struct ProducedHistory {
+		uint16_t production; ///< Total produced
+		uint16_t transported; ///< Total transported
+
+		uint8_t PctTransported() const
+		{
+			if (this->production == 0) return 0;
+			return  ClampTo<uint8_t>(this->transported * 256 / this->production);
+		}
+	};
+
+	struct ProducedCargo {
+		CargoID cargo; ///< Cargo type
+		uint16_t waiting; ///< Amount of cargo produced
+		uint8_t rate; ///< Production rate
+		uint8_t pct_transported; ///< Percentage transported last month
+		std::array<ProducedHistory, 2> history; ///< History of cargo produced and transported
+
+	};
+
+	struct AcceptsCargo {
```

Comment:
Conceptually, shouldn't this be called `AcceptedCargo`? After all, the waiting cargo is already accepted, and last_accepted...

Reply:
Reasonable, and fits between with `Produced` which doesn't really work as `Produces` either.

Reply:
For consistency that's probably best.